charts/gardener/gardenlet/charts/runtime/templates/clusterrole-gardenlet.yaml

```
@@ -0,0 +1,293 @@
+{{- if .Values.global.gardenlet.enabled }}
+# ClusterRole defines the required permissions for the gardenlet in the seed
+# For now, we provide escalation privileges required by gardener resoruce manager.
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:system:gardenlet
+  labels:
+    gardener.cloud/role: gardenlet
+    app: gardener
+    role: gardenlet
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+rules:
```

https://github.com/gardener/gardener/pull/3811 was recently merged and it lists all Seed nodes to decide on seed-admission-controller Deployment replicas.

```suggestion
rules:
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - list
  - watch
```

```
time="2021-04-09T13:16:29+03:00" level=error msg="seed is not yet ready: condition \"Bootstrapped\" has invalid status False (expected True) due to BootstrappingFailed: 1 error occurred:\n\t* task \"Deploying gardener-seed-admission-controller\" failed: nodes is forbidden: User \"system:serviceaccount:garden:gardenlet\" cannot list resource \"nodes\" in API group \"\" at the cluster scope\n\n" operation=reconcile shoot=garden-foo/aws-local
```